src/frontend/components/PublicLiveDashboard/index.tsx

```diff
@@ -22,14 +25,30 @@ export const PublicLiveDashboard = ({
 }: PublicLiveDashboardProps) => {
   const intl = useIntl();
   const [isReadyLive, setIsReadyLive] = useState(false);
-
-  useAsyncEffect(async () => {
-    if (!isReadyLive) {
-      await initWebinarContext(live, intl.locale);
-      initVideoWebsocket(live);
-      setIsReadyLive(true);
-    }
-  }, [isReadyLive, live]);
+  const anonymousId = useMemo(() => getOrInitAnonymousId(), []);
+  const queryClient = useQueryClient();
+  const setLiveSession = useLiveSession((state) => state.setLiveSession);
+  useLiveSessions(
```

Comment:
I'm not a big fan of multiple hooks sharing the same name (even if there are quite a few like that in the project). 
What do you think about renaming this hook `useLiveSessionQuery` ?

Reply:
+1